CHANGELOG.md

```diff
@@ -59,7 +59,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 * (types) [#19759](https://github.com/cosmos/cosmos-sdk/pull/19759) Align SignerExtractionAdapter in PriorityNonceMempool Remove.
 * (client) [#19870](https://github.com/cosmos/cosmos-sdk/pull/19870) Add new query command `wait-tx`. Alias `event-query-tx-for` to `wait-tx` for backward compatibility.
 * (genutil) [#19971](https://github.com/cosmos/cosmos-sdk/pull/19971) Allow manually setting the consensus key type in genesis
-* (client) [#20266](https://github.com/cosmos/cosmos-sdk/pull/20266) Ability to override short description in AutoCLI generated top-level commands.
+* (client/v2) [#20266](https://github.com/cosmos/cosmos-sdk/pull/20266) Ability to override short description in AutoCLI generated top-level commands.
```

Comment:
this should be under client/v2/changelog :D

Reply:
https://github.com/cosmos/cosmos-sdk/pull/20266/commits/53c0fec4d4c8912c38892a7b7a649d088c90af94